lib/src/markdown_processor.dart

```diff
@@ -880,6 +967,60 @@ md.Node _makeLinkNode(String codeRef, Warnable warnable,
   return md.Element.text('code', textContent);
 }
 
+MatchingLinkResult _getMatchingLinkElement(Warnable warnable, String codeRef,
+    List<ModelCommentReference> commentRefs) {
+  MatchingLinkResult result, resultOld, resultNew;
+  // Do a comparison between result types only if the warnings for them are
+  // enabled, because there's a significant performance penalty.
```

Comment:
Haha I was wondering exactly this question 2 minutes before I read this comment.